xcodeproj/internal/bazel_integration_files/copy_dsyms.sh

```diff
@@ -63,6 +63,9 @@ if [[ -n "${BAZEL_OUTPUTS_DSYM:-}" ]]; then
     $(xargs -n1 <<< "$BAZEL_OUTPUTS_DSYM") \
     "$TARGET_BUILD_DIR"
 
+  # Works around an rsync issue causing "Permission denied" errors
+  chmod +w "$TARGET_BUILD_DIR"
```

Comment:
Do we want to do this unconditionally or per OS version? Lacking some context about the previous PR adding `--perms` so wanted to confirm

Reply:
cc @luispadron @brentleyjones when you're available I'd appreciate if you could clarify please 🙇

Reply:
I don't think we want a version check.